backends/bevy_picking_ui/src/lib.rs

```diff
@@ -65,7 +65,7 @@ pub struct NodeQuery {
 /// we need for determining picking.
 pub fn ui_picking(
     pointers: Query<(&PointerId, &PointerLocation)>,
-    cameras: Query<(Entity, &Camera, Option<&UiCameraConfig>)>,
+    cameras: Query<(Entity, &Camera, Option<(&Node, &Visibility)>)>,
```

Comment:
I did not have a chance to test if this is the proper way of querying for the root node.

https://bevyengine.org/news/bevy-0-13/#camera-driven-ui

I saw they suggest using NodeBundle to toggle UI which includes Node and Visibility

Reply:
After having more time to review, this is wrong.

There are now `DefaultUiCamera` and `TargetCamera`

potential options:

`Option<&TargetCamera>`

`(With<Camera>, With<IsDefaultUiCamera>)>`

Reply:
notes:

[TargetCamera](https://docs.rs/bevy/latest/bevy/prelude/struct.TargetCamera.html):

> Indicates that this root [Node](https://docs.rs/bevy/latest/i686-pc-windows-msvc/bevy/ui/struct.Node.html) entity should be rendered to a specific camera. UI then will be layed out respecting the camera’s viewport and scale factor, and rendered to this camera’s [bevy_render::camera::RenderTarget](https://docs.rs/bevy/latest/i686-pc-windows-msvc/bevy/render/camera/enum.RenderTarget.html).
>
>
> Setting this component on a non-root node will have no effect. It will be overriden by the root node’s component.
> 
> Optional if there is only one camera in the world. Required otherwise.


[IsDefaultUiCamera](https://docs.rs/bevy/latest/bevy/ui/struct.IsDefaultUiCamera.html):

> Marker used to identify default cameras, they will have priority over the [PrimaryWindow](https://docs.rs/bevy/latest/i686-pc-windows-msvc/bevy/window/struct.PrimaryWindow.html) camera.
>
>This is useful if the [PrimaryWindow](https://docs.rs/bevy/latest/i686-pc-windows-msvc/bevy/window/struct.PrimaryWindow.html) has two cameras, one of them used just for debug purposes and the user wants a way to choose the default [Camera](https://docs.rs/bevy/latest/i686-pc-windows-msvc/bevy/render/camera/struct.Camera.html) without having to add a [TargetCamera](https://docs.rs/bevy/latest/i686-pc-windows-msvc/bevy/ui/struct.TargetCamera.html) to the root node.
>
> Another use is when the user wants the Ui to be in another window by default, all that is needed is to place this component on the camera

I think I understand now and see the comment describing the usage in `ui_picking`. 
> // Find the topmost bevy_ui camera with the same target as this pointer.

Attempting to update..